cms/envs/common.py

```diff
@@ -1041,6 +1041,8 @@
     'django_comment_common',
     'edraak_misc',
 
+    'edraak_marketing_email.apps.EdraakMarketingEmailConfig',
+
```

Comment:
Why do you think we would need it in the `cms`? I don't think we do.

```suggestion
```

Reply:
@OmarEdraak , no it is not needed. But I suggested adding it so pass CMS tests. They were failing because of importing the new app in `common/djangoapps/student/views/management.py`.

Reply:
Thanks @shadinaif! Looks good to me then.

We should move it into the future (https://github.com/Edraak/edraak-platform/issues/162) into `common`. That's a work for another day.

@Fahminajjar could you please test the current version of your pull request? If it's good, please feel free to merge.